src/admin-events/admin-events.js

```diff
@@ -0,0 +1,96 @@
+const { Finding, FindingSeverity, FindingType } = require('forta-agent');
+const { getAbi, getAddress } = require('@uma/contracts-node');
+const ethers = require('ethers');
+
+// load config files
+const config = require('../../agent-config.json');
```

Comment:
Optional: You are only interested in the `umaEverestId` value from the `agent-config.json` file, so you could use destructuring here:
```suggestion
const { umaEverestId } = require('../../agent-config.json');
```
Then line 56 would change to:
```
    everestId: umaEverestId
```

Reply:
Done